tests/system/src/test/java/test/robot/javafx/stage/StageMixedSizeTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2024 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Copyright should be `2024, 2025,` when you're updating an older file

Reply:
Fixed